carbon-design-system/carbon-addons-iot-react
# fix(customcard): rendering of custom card was incorrect

Closes #

**Summary**

- I broke the custom card rendering from a function here #2270 , because only the first array child was an actual function now that I'm passing the additional content fields (to support resizing)

**Change List (commits, features, bugs, etc)**

- fix(Card.jsx): check the children to see if it's an array and call the first function in an array to render the custom content
- test(DashboardEditor): update the custom card story to actually render custom content from a function to catch this kind of regression in the future
- fix(package.json): react-draggable isn't bringing in fbjs as a dependency so local yarn links fail.  I'm specifying this as a direct dependency of carbon-addons-iot-react now to make sure the local yarn link can build.

**Acceptance Test (how to verify the PR)**

- Verify that the custom card content renders from a function in the DashboardEditor story called `with initialValues`: https://next.carbon-addons-iot-react.com/?path=/story/watson-iot-experimental-%E2%98%A2%EF%B8%8F-dashboardeditor--with-initial-value

**Regression Test (how to make sure this PR doesn't break old functionality)**
- Verify that the existing table cards and image cards render correctly in the stories (as they use functions to render their children)
